dell/precision/5560/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, ... }:
+{
+  imports = [
+    ../../../common/pc/laptop
+    ../../../common/pc/laptop/ssd
+    ../../../common/cpu/intel
+    ../../../common/gpu/intel/tiger-lake
+    ../../../common/gpu/nvidia/turing
+  ];
+
+  hardware.enableRedistributableFirmware = lib.mkDefault true;
+
+  boot = {
+    blacklistedKernelModules = [ "nouveau" ];
+    kernelModules = [ "kvm-intel" ];
```

Comment:
nixos-generate-config already detect this:

```suggestion
```

Reply:
fixed